tests/cli/test_cli_install.py

```diff
@@ -57,11 +57,17 @@ def test_find_conflicts_called_once(
         "conda.resolve.Resolve.find_conflicts",
         side_effect=UnsatisfiableError(bad_deps, strict=True),
     )
-
-    with tmp_env("python=3.9") as prefix:
+    channels = (
+        "--repodata-fn",
+        "current_repodata.json",
+        "--override-channels",
+        "-c",
+        "defaults",
+    )
```

Comment:
This kind of change reduces the runtime to ~20s by simply reducing the problem size. Many tests in test_create could benefit from a toy-sized channel tailored to reproduce the problem.